src/AWSCredentials.jl

```diff
@@ -254,27 +254,62 @@ using IniFile
 dot_aws_credentials_file() = get(ENV, "AWS_CONFIG_FILE",
                                  joinpath(homedir(), ".aws", "credentials"))
```

Comment:
It seems based on the [current doc](https://docs.aws.amazon.com/cli/latest/topic/config-vars.html#the-shared-credentials-file) that this should be changed to:
```julia
dot_aws_credentials_file() =
    get(ENV, "AWS_SHARED_CREDENTIALS_FILE",
        joinpath(homedir(), ".aws", "credentials")) 
```

Reply:
Oops, I forgot to change that. Thanks!